examples/libp2p/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.12)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-libp2p)
+
+hunter_add_package(libp2p)
+find_package(libp2p CONFIG REQUIRED)
```

Comment:
Please add a duplicate find-pacjage call to have a check if the created package behaves nicely